fsharp-backend/tests/testfiles/internal.tests

```diff
@@ -184,3 +184,67 @@ DarkInternal.unlockedDBs Test.getCanvasID = [1]
 [test.unlocked, one but locked] with DB X
 (let _ = DB.set_v1 { x = "str" } "test" X in
  DarkInternal.unlockedDBs Test.getCanvasID) = []
+
+
+// ---------------
```

Comment:
Maybe I'm not looking in the right place but I don't see a test for fetching all the deploys.

Reply:
The tests in `internal.tests` call the 'fetch all deploys' (`DarkInternal.staticAssetsDeploys`) fn in various scenarios, and check its simple length, or map to the `status` and just check _that_ resultant list.

The result of that call returns dynamic data (`deployHash`, `url`, `lastDate`) that aren't quite useful to test here beyond checking count and `status` data. If you have other tests to recommend or ways of capturing these fields reasonably in tests, I'm open to ideas!